diff --git a/src/diff/index.js b/src/diff/index.js
index c2390f6..f882dc6 100644
--- a/src/diff/index.js
+++ b/src/diff/index.js
@@ -1,4 +1,4 @@
-import { TEXT_NODE, EMPTY_OBJ, EMPTY_ARR } from '../constants';
+import { TEXT_NODE, FRAGMENT, EMPTY_OBJ, EMPTY_ARR } from '../constants';
 import { assign } from '../util';
 import { Component, enqueueRender } from '../component';
 import { createVNode /*, reclaimVNode*/ } from '../create-element';
@@ -267,7 +267,13 @@ export function diff(dom, parent, newTree, oldTree, context, isSvg, append, exce
 		// 	// console.trace('diffing '+c.id);
 		// 	console.log('diffing '+c.id, vnode, prev);
 		// }
-		c.base = diff(dom, parent, vnode, prev, context, isSvg, append, excessChildren);
+		if(vnode.type===FRAGMENT) {
+			let prevChild = prev ? prev.props.children: [];
+			diffChildren(parent, vnode.props.children, prevChild, context, isSvg, excessChildren);
+			return;
+		} else {
+			c.base = diff(dom, parent, vnode, prev, context, isSvg, append, excessChildren);
+		}
 		c._dirty = false;
 		// newTree.tag.$precache = c.base;
 		c._vnode = newTree;
@@ -646,4 +652,4 @@ function createComponent(Ctor, props, context) {
 /** The `.render()` method for a PFC backing instance. */
 function doRender(props, state, context) {
 	return this.constructor(props, context);
-}
\ No newline at end of file
+}
